__tests__/app.test.js

```diff
@@ -13,6 +13,32 @@ afterAll(() => {
     db.end();
 })
 
+describe("GET /api", () => {
+    it("returns a JSON object that includes a description, query & example response", () => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.endpoints).toHaveProperty("GET /api/topics")
+                const endpoints = response.body.endpoints
+                const endpointArr = Object.values(endpoints)
+                endpointArr.forEach((endpoint) => {
```

Comment:
If we are iterating through an array in our test file we need to do a length check, if an empty array was passed through here this test would pass.